Clover/app/src/main/java/org/floens/chan/core/presenter/ThreadPresenter.java

```diff
@@ -698,6 +707,9 @@ private void showPostInfo(Post post) {
         }
 
         // TODO(multi-site) get this from the timestamp
+        Date date = new Date () ;
+        date.setTime((long)post.time*1000);
+        text.append(new StringBuilder("Date: " + date.toString()));
```

Comment:
instead of creating a new `StringBuilder` object,  you could do something like `text.append("Date: ").append(date.toString()));`